File: src/test/java/no/utgdev/diy/ex5/Ex5Config.java
package no.utgdev.diy.ex5;

//REMOVE import no.utgdev.diy.annotations.Bean;

public class Ex5Config {
    //REMOVE @Bean(name = "min-controller")
    public Controller1 controller() {
        return new Controller1();
    }

    //REMOVE @Bean(name = "min-controller2")
    public Controller2 controller2() {
        return new Controller2();
    }

    //REMOVE @Bean(name = "hello-dao")
    public HelloDao helloDao() {
        return new HelloDao("Greetings");
    }

    //REMOVE @Bean(name = "hello-dao2")
    public HelloDao helloDao2() {
        return new HelloDao("Welcome");
    }
}

File: src/test/kotlin/no/utgdev/diy/ex5/Ex5Config.kt
package no.utgdev.diy.ex5

import no.utgdev.diy.annotations.Bean;

class Ex5Config {
    @Bean(name = "min-controller")
    fun controller(): Controller1 {
        return Controller1()
    }

    @Bean(name = "min-controller2")
    fun controller2(): Controller2 {
        return Controller2()
    }

    @Bean(name = "hello-dao")
    fun helloDao(): HelloDao {
        return HelloDao("Greetings")
    }

    @Bean(name = "hello-dao2")
    fun helloDao2(): HelloDao {
        return HelloDao("Welcome")
    }
}